### PR TITLE
Add test for local quorum queries while a node is bootstrapping

### PR DESCRIPTION
**Pull request by aboudreault**
Add test for local quorum queries while a node is bootstrapping

### CODE REVIEW

**Comment by ptnapoleon** [comment]
@aboudreault What's the status on this?

**Comment by aboudreault** [comment]
I tried to finish that last week... but now getting issues related the FlushWriter in 2.0... should be good to work on this this week.

**Comment by ptnapoleon** [comment]
No rush, just wanted to make sure we still wanted to keep this PR open. We have a lot built up

**Comment by aboudreault** [comment]
ready for review

File: bootstrap_test.py [hunk 1]
```diff
@@ -199,3 +203,69 @@ def manual_bootstrap_test(self):
...
+        node3 = new_node(cluster, data_center='dc2')
+        node3.start(no_wait=True)
+        time.sleep(3)
+
+        with tempfile.TemporaryFile(mode='w+') as tmpfile:
+            if version < "2.1":
+                node1.stress(['-o', 'insert', '-n', '500000', '-t', '5', '-e', 'LOCAL_QUORUM', '-K', '2'],
+                             stdout=tmpfile, stderr=subprocess.STDOUT)
+            else:
+                node1.stress(['user', 'profile='+stress_config.name, 'ops(read=1)',
```
Comment (ptnapoleon) [inline bootstrap_test.py]: It looks like we're reading here, but in the `if version < "2.1":` block, we're inserting. Which is correct?

Comment (aboudreault) [inline bootstrap_test.py]: no, fixing

Comment (aboudreault) [inline bootstrap_test.py]: fixed

**Comment by ptnapoleon** [comment]
LGTM, +1

**Pull request merged**